mxcube3/routes/Queue.py

```diff
@@ -309,6 +311,7 @@ def executeEntryWithId(nodeId):
                         mxcube.queue.queue_hwobj.execute_entry(childEntry)
                         time.sleep(1) # too fast to synch properly signals, it should not be a problem with real stuff
                     except Exception:
+                        print sys.exc_info()
```

Comment:
what about removing this `print` and replacing `logging.error` below with `logging.exception` ?